### PR TITLE
feat: remove EventEmitter from 'event' module

### PR DESCRIPTION
**Pull request by nightink**
remove EventEmitter from 'event' module
set package engines field

nodejs/node#2921
https://github.com/nodejs/node/commit/6ed861dd7feea33224d1cac8e64010b45974ae51

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,10 @@
     "mock",
     "test"
   ],
+  "engines": {
+    "iojs": ">=1.0.0",
```
Comment (fengmk2) [inline package.json]: just node, iojs was deped.

File: package.json [hunk 2]
```diff
@@ -47,6 +47,10 @@
...
+    "node": ">=0.11.0"
```
Comment (fengmk2) [inline package.json]: >=0.12.0

**Comment by nightink** [comment]
done!

**Comment by fengmk2** [comment]
+1

**Pull request merged**